app/routes/agenda-items/agenda-item.ts

```diff
@@ -13,7 +13,9 @@ export default class AgendaItemRoute extends Route {
   @service declare keywordStore: KeywordStoreService;
 
   async model(params: DetailParams) {
-    const agendaItem = await this.store.findRecord('agenda-item', params.id);
+    const agendaItem = await this.store.findRecord('agenda-item', params.id, {
+      include: 'handled-by',
+    });
 
     // wait until sessions are loaded
     const sessions = await agendaItem.sessions;
```

Comment:
I put the comment here, but concern line 54. `const vote = (await agendaItemHandling?.hasVotes)?.toArray().shift();` is not needed here anymore.

Reply:
Maybe I misunderstand by `this.model.vote` is used in the `hasVotes` getter in the controller, so I think it's still needed?

Reply:
If instead we use `this.model.vote` directly in template should it load data when we open the accordion?

Reply:
Yea, but I really dislike loading data through the template. It makes things hard to follow. If you don't want to load it in the model hook I would use a resource instead. That would also allow us to display loading states, instead of the glitchy rendering that happens otherwise.

It also seems unrelated to what this PR is doing, so I think we can do that in a seperate one? (regardless if we load through the template or through a different setup).

Reply:
:+1: